apps/prairielearn/src/ee/institution/admin/courses.html.ts

```diff
@@ -0,0 +1,59 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { type Course, type Institution } from '../../../lib/db-types';
+
+export function InstitutionAdminCourses({
+  institution,
+  courses,
+  resLocals,
+}: {
+  institution: Institution;
+  courses: Course[];
+  resLocals: Record<string, any>;
+}) {
+  return html`
+    <!DOCTYPE html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/head')%>", {
+          ...resLocals,
+          navPage: 'institution_admin',
+          pageTitle: 'Courses',
+        })}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/navbar') %>", {
+          ...resLocals,
+          institution,
+          navbarType: 'institution',
+          navPage: 'institution_admin',
+          navSubPage: 'courses',
+        })}
+        <main class="container mb-4">
+          <div class="table-responsive">
+            <table class="table table-hover table-striped table-bordered table">
+              <thead>
+                <tr>
+                  <th>Name</th>
+                </tr>
+              </thead>
+              <tbody>
+                ${courses.map((course) => {
+                  return html`
+                    <tr>
+                      <td>
+                        <a href="/pl/institution/${institution.id}/admin/course/${course.id}">
+                          ${course.short_name ?? '—'}: ${course.title ?? '—'}
```

Comment:
I've been using mini-components like `CourseNameAndTitle(course)` for things like this, to maintain consistency across pages. Totally not necessary, of course.